app/main.py

```diff
@@ -5,7 +5,7 @@ def get_human_age(cat_age: int, dog_age: int) -> list:
 
 
 def convert_to_human(
-        animal_age: int, first_year: int, second_year: int, each_year: int
+    animal_age: int, first_year: int, second_year: int, each_year: int
```

Comment:
There is a syntax error in the function parameter list. The division operator `/` is incorrectly used in the parameter declaration `each_year: int /`. It should be a comma `,` instead of a division operator to separate the parameters.